test/functional/fixtures.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var node = require('../node');
+
+var faker = require('faker');
+
+var maximumString = node.randomString.generate(64);
+var maximumStringPlus1 = node.randomString.generate(64 + 1);
+
+var testCases = [
```

Comment:
Do we have to test all the cases as a part of this PR? Almost the same thing is already in place: `test/common/typesRepresentatives`

Reply:
I plan to review all the common parts in a different PR.